public/assets/js/ability-scores.js

```diff
@@ -64,35 +64,52 @@ async function postAbilityScores (heroID, name, score, modifier) {
             'Content-Type': 'application/json'
         }
     })
-    if (response.ok) {
-        console.log("yay")
-    document.location.reload();
-    } else {
-    alert(response.statusText);
-    }
+    
+    //-- API Response SUCCESS
+    if      (response.ok) document.location.reload();
+    
+    //-- API Response FAIL
+    else    alert(response.statusText);
 }
     
 async function createAbilityScores () {
+    /** Manage calculating abilities scores when user clicks "Calculate" button.
```

Comment:
Simplified this function, added more complete logic notes, verified integrity, and re-enabled this function to fix the issue with ability scores NOT generating.